src/test/resources/credentials.properties

```diff
@@ -1,5 +1,6 @@
 username=lilia_0801@ukr.net
 password=Test123456
+wait=10
 userId=31xqqvchtblksf3v2jevchgppzn4
 grantType=refresh_token
 clientId=c9e017d2fd6e4e9f992e7430a3ddd38f
```

Comment:
Usually files with secrets you can send directly to reviewer. It's a great mistake to push any secrets into public repo (and sure private repos too))).